core/src/components/AppMenuEntry.vue

```diff
@@ -61,10 +61,10 @@ defineProps<{
 		text-align: center;
 		bottom: 0;
 		left: 50%;
+		top: 50%;
```

Comment:
This one is the only problematic place I think because with unread notification it looks like this now:
![Screen Shot 2024-07-31 at 15 31 57](https://github.com/user-attachments/assets/6ab2b590-3860-4fcb-a40a-6f5698aa7807)

With this:

```suggestion
		top: 60%;
```

It would be:
![Screen Shot 2024-07-31 at 15 31 38](https://github.com/user-attachments/assets/9b2db148-98cf-474a-95a5-765fdd427b2a)

Reply:
![Peek 31-07-2024 17-19](https://github.com/user-attachments/assets/8d94f7ea-d4b3-4bdc-a116-85b68cb010e7)